robottelo/ui/base.py

```diff
@@ -163,14 +202,9 @@ def delete_entity(self, name, really, name_locator, del_locator,
         # Make sure that element is really removed from UI. It is necessary to
         # verify that fact few times as sometimes 1 second is not enough for
         # element to be actually deleted from DB
+        self.is_entity_delete = True
         for _ in range(3):
```

Comment:
NACK on that comment as we will never experience any situation like that and will get variable into initial case in case of any failure
'negative delete test, which tries to delete and then searches for entity to verify it wasn't actually deleted?' is already covered here and will put variable into False state (`really` variable help us a lot)

Basically, don't want to introduce redundant block and make code less readable